_source/_posts/2022-08-12-ci-cd-microservices-circleci-spinnaker.md

```diff
@@ -0,0 +1,949 @@
+---
+layout: blog_post
+title: "CI/CD with CircleCI and Spinnaker for JHipster Microservices"
```

Comment:
@havi The coolest parts of this post are CircleCI and Spinnaker. Both have good SEO as keywords on their own, but not with much else. If we want to shorten the title, I'd be cool with dropping "for JHipster Microservices".

Reply:
When we syndicate to dev.to, we could add back in the JHipster microservices part.